tests/fps_tests.py

```diff
@@ -704,6 +706,184 @@ def test_expected_esld(self):
         self.assertEqual(loaded_sets, expected_sets)
         self.assertEqual(fp.error_list, [])
 
+class TestFindDiff(unittest.TestCase):
+    def test_unchanged_sets(self):
+        old_sets = {
+            'https://primary.com': 
+            FpsSet(
+                    primary="https://primary.com", 
+                    associated_sites=None,
+                    service_sites=None,
```

Comment:
```suggestion
```

Same comment elsewhere.

Reply:
Made this change, but this required a change to the FpsSet class in order to work. I will follow up on this in a future PR to make a similar change to ccTLDs, as that might be a little more complicated.